rflx/generator/allocator.py

```diff
@@ -134,93 +155,96 @@ def _create_ptr_subtypes(self) -> UnitPart:
                         )
                     ],
                 )
-                for size in set(self._slot_sizes.values())
+                for size in {slot.size for slot in slots}
```

Comment:
A Python set is unordered, so this code could lead to a different result each time it is used for code generation. This issue could be prevented by using `sorted` on the set.

Reply:
Fixed in dc8838412927382bf2e36f4042733b98bab41d90